<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMakeLayerDlg</class>
 <widget class="QDialog" name="CMakeLayerDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create New Map</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Orgin Map</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="c_origin">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>New Map Type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="c_layer_type"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>New Map Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="c_layer_name"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QCheckBox" name="c_keeporiginal">
       <property name="text">
        <string>Retain original map</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="c_push_values">
       <property name="text">
        <string>Copy attributes to new map</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="c_remove_stubs">
         <property name="text">
          <string>Remove axial stubs less than</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="c_percentage">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>% of line length</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="c_ok">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>c_layer_type</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>CMakeLayerDlg</receiver>
   <slot>OnSelchangeLayerType(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>c_remove_stubs</sender>
   <signal>clicked(bool)</signal>
   <receiver>CMakeLayerDlg</receiver>
   <slot>OnRemoveStubs(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>c_ok</sender>
   <signal>clicked()</signal>
   <receiver>CMakeLayerDlg</receiver>
   <slot>OnOK()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>49</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>c_cancel</sender>
   <signal>clicked()</signal>
   <receiver>CMakeLayerDlg</receiver>
   <slot>OnCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnSelchangeLayerType(int)</slot>
  <slot>OnRemoveStubs(bool)</slot>
  <slot>OnOK()</slot>
  <slot>OnCancel()</slot>
 </slots>
</ui>
